util/wct.sh

```diff
@@ -21,4 +21,9 @@ if [ "$USE_FRAME_BUFFER" == "true" ]; then
 fi
 
 cd webapp
-npm test
+if [ "$UID" == "0" ]; then
+  # Used in .github/actions/make-in-docker/Dockerfile
```

Comment:
I'm somewhat concerned we're going to end up sprinkling `if UID == 0 { sudo -u browser foo } else { foo }` everywhere in our scripts (or worse, miss adding it and break GH actions). I don't immediately have a suggestion on how to address this, but something to think about.

Reply:
ditto